### ISSUE TITLE
Can sharktrust work entirely offline?

### ISSUE THREAD

**Issue by jax79sg** [RealTimeLogic/SharkTrust#1]
Sharktrust uses Let's Encrypt, which means it is necessary to communicate to the Internet. Is it possible for Sharktrust to be used in a setup where there is absolutely no internet access. The only way is to transfer data via usb drives.

**Comment by UmairAsghar-max** [comment]
@jax79sg and @Telemechanics, any updates to this question

**Comment by surfskidude** [comment]
No, and you do not have an option with any solution to be offline.
